src/analysis/rMTA/MTA.m

```diff
@@ -0,0 +1,219 @@
+%                     Backward and Unchanged (+1;0;-1). This is calculated
+%                     from the rules and differential expression analysis.
+%    Vref:            Reference flux of the source state.
+%
+% OPTIONAL INPUT:
+%    alpha:           Numeric value or array. Parameter of the quadratic
+%                     problem (default = 0.66)
+%    epsilon:         Numeric value or array. Minimun perturbation for each
+%                     reaction (default = 0)
+% 
+% OPTIONAL INPUT (name-value pair):
+%    rxnKO            Binary value. Calculate knock outs at reaction level 
+%                     instead of gene level. (default = false)
+%    timelimit        Time limit for the calculation of each knockout.
+%                     (default = inf)
+%    SeparateTranscript - Character used to separate
+%                         different transcripts of a gene. (default = '')
+%                         Example: SeparateTranscript = ''
+%                                   gene 10005.1    ==>    gene 10005.1
+%                                   gene 10005.2    ==>    gene 10005.2
+%                                   gene 10005.3    ==>    gene 10005.3
+%                                  SeparateTranscript = '.'
+%                                   gene 10005.1
+%                                   gene 10005.2    ==>    gene 10005
+%                                   gene 10005.3
+%    numWorkers        Integer: is the maximun number of workers
+%                      used by the solver. 0 = automatic, 1 = sequential,
+%                         >1 = parallel. (default = 0)
+%    printLevel        Integer. 1 if the process is wanted to be shown
+%                      on the screen, 0 otherwise. (default = 1)
+%
+% OUTPUTS:
+%    Outputs are cell array for each alpha (one simulation by alpha). It
+%    there is only one alpha, content of cell will be returned
+%    TSscore:         Transformation score by each transformation
+%    deletedGenes:    The list of genes/reactions removed in each knock-out
+%    Vres:            Matrix of resulting fluxes
+%
+% .. Authors:
+%       - Luis V. Valcarcel, 03/06/2015, University of Navarra, CIMA & TECNUN School of Engineering.
+% .. Revisions:
+%       - Luis V. Valcarcel, 26/10/2018, University of Navarra, CIMA & TECNUN School of Engineering.
+%       - Francisco J. Planes, 26/10/2018, University of Navarra, TECNUN School of Engineering.
+
+
+%% Check the input information
+p = inputParser;
+% check requiered arguments
+addRequired(p, 'model');
+addRequired(p, 'rxnFBS', @isnumeric);
+addRequired(p, 'Vref', @isnumeric);
+% Check optional arguments
+addOptional(p, 'alpha', 0.66, @isnumeric);
+addOptional(p, 'epsilon', 0, @isnumeric);
+% Add optional name-value pair argument
+addParameter(p, 'rxnKO', false);
+addParameter(p, 'timelimit', inf);
+addParameter(p, 'SeparateTranscript', '');
+addParameter(p, 'numWorkers', 0);
+addParameter(p, 'printLevel', 1);
```

Comment:
You could also add validation functions (similar to the optional arguments) here, to give more precise errors, when something wrong is supplied.